cpp/tests/io/nested_json_test.cpp

```diff
@@ -245,12 +246,13 @@ TEST_F(JsonTest, TokenStream)
   using cuio_json::SymbolOffsetT;
   using cuio_json::SymbolT;
 
-  constexpr std::size_t single_item = 1;
-
   // Prepare cuda stream for data transfers & kernels
   rmm::cuda_stream stream{};
   rmm::cuda_stream_view stream_view(stream);
```

Comment:
similar suggestion from another PR
```suggestion
  constexpr auto stream = cudf::default_stream_value
```
replace all stream_view with this stream.

Reply:
Oh that makes sense, thanks! 🙏